sdk/JavaScript/TestFramework/js/testFramework.js

```diff
@@ -514,31 +547,37 @@ function createZumoNamespace() {
         TSSkipped: TSSkipped,
         AllTestsGroupName: AllTestsGroupName,
         AllTestsUnattendedGroupName: AllTestsUnattendedGroupName,
-        constants: {
+        constants: {
             CLIENT_VERSION_KEY: ClientVersionKey,
             SERVER_VERSION_KEY: ServerVersionKey,
-            RUNTIME_FEATURES_KEY: RuntimeFeaturesKey
+            RUNTIME_FEATURES_KEY: RuntimeFeaturesKey
         },
-        runtimeFeatureNames: {
+        runtimeFeatureNames: {
             STRINGREPLACE: 'stringReplace',
-            DotNetRuntime_Only: 'dotnetRuntimeOnly',
+            DotNetRuntime_Only: 'dotNetRuntimeOnly',
             NodeRuntime_Only: 'nodeRuntimeOnly',
             INT_ID_TABLES: 'intIdTables',
             STRING_ID_TABLES: 'stringIdTables',
-            NH_PUSH_ENABLED: 'nhPushEnabled'
+            NH_PUSH_ENABLED: 'nhPushEnabled'
         },
         Test: ZumoTest,
         Group: ZumoTestGroup,
         tests: {},
-        util: {
+        util: {
             createSeparatorTest: createSeparatorTest,
             compare: compareValues,
             traceResponse: traceResponse,
             dateToString: dateToString,
             randomDate: randomDate,
-            globalTestParams: {}
-        }
-    };
+            globalTestParams: {}
+        },
+        assert: {
+            areEqual: assertAreEqual,
+            areNotEqual: assertAreNotEqual,
+            isTrue: assertIsTrue,
+            isFalse: assertIsFalse
+        }
+    };
```

Comment:
testFramework.js is getting long! These "namespaces" should really be contained in other files to improve discoverability, reuse, etc. No action for now, but if we continue to make changes to this file, we should split them out.

We should also really be using an assertion library such as chai, there is no need to reimplement assertions, but not worth the effort to introduce at this stage. If we start to need other more complex assertions like deepEqual, we should revisit this.